tests/test_other.py

```diff
@@ -61,6 +61,19 @@ def decorated(self):
   return decorated
 
 
+def check_python3_version():
```

Comment:
I'd call this `is_python3_version_supported` or something.
Functions named "check" I expect to be void-returning, and signal their success/failure via exceptions or some other mechanism. Either that or they're ambiguously named (does `check_foo()` return true if there's something wrong with foo, or if foo is ok?)